data/discovery/field/citizens-advice-bureau-contact-region.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: citizens-advice-bureau-contact-region
+phase: discovery
+register: citizens-advice-bureau-contact-region
+text: Contact region for Citizens Advice Bureaux.
```

Comment:
`Contact region of a Citizens Advice Bureau in England or Wales`

Reply:
I have updated this attribute